docker-for-mac/release-notes.md

```diff
@@ -16,10 +16,25 @@ This page contains information about the new features, improvements, known issue
 > Starting with Docker Desktop 3.0.0, Stable and Edge releases are combined into a single release stream for all users. Updates to Docker Desktop will now be available automatically as delta updates from the previous version. This means, when there is a newer version of Docker Desktop, it will be automatically downloaded to your machine. All you need to do is to click **Update and restart** from the Docker menu to install the latest update.
 {: .important }
 
+## Docker Desktop 3.2.2
+2021-03-15
+
+> [Download](https://desktop.docker.com/mac/stable/amd64/Docker.dmg)
+
+### Bug fixes and minor changes
+
+- Allow exposing port 53 (DNS) without clashing with an internal DNS server. Fixes [docker/for-mac#5416](https://github.com/docker/for-mac/issues/5416).
+- Fixed an issue related to running iTerm that has no window open [docker/roadmap#98](https://github.com/docker/roadmap/issues/98#issuecomment-791927788)
+- Fixed an issue related to high CPU consumption and frozen UI when network connection lost [for-win/#10583](https://github.com/docker/for-win/issues/10563)
+
+### Known issue
+
+The new Apple Virtualization framework uses port 53 (DNS) when Docker Desktop is started. This port cannot be used to bind a container port to the host.
```

Comment:
We don't need to document this as the virtualization framework is experimental only. (We should put it on the M1 page, but not here).

Reply:
Alright, I'll remove it here and create a separate PR.

Reply:
Updated and created #12499